policyengine_us/parameters/gov/states/mi/tax/income/deductions/standard/birth_year.yaml

```diff
@@ -0,0 +1,22 @@
+description: Michigan provides a standard deduction addition for recipients born after this year. 
+values:
+  2017-01-01: 1941
+  2018-01-01: 1942
+  2019-01-01: 1943
+  2020-01-01: 1944
+  2021-01-01: 1945
+  2022-01-01: 1946
+metadata:
+  unit: year
+  label: Michigan standard deduction addition age max threshold
+  reference:
+    - title: Michigan Legal Code Section 206.30 - (9)(b)&(e)
```

Comment:
Similar to above, can we simplify the logic by more closely matching the legal code?

Reply:
what do you mean by simplify the logic?

Reply:
Since the legal code doesn't specify a different year value for every year, can we define the logic analogously to the law and have fewer parameters?

Reply:
do you mean just using 1946 for all years?

Reply:
I haven't read the law in detail. But I don't see how the current code reflects the law. Please let me know if I'm missing it.

Reply:
it only mentioned the year 1946 like this 

>   (9) In determining taxable income under this section, the following limitations and restrictions apply:
  (a) For a person born before 1946, this subsection provides no additional restrictions or limitations under subsection (1)(f).
  (b) Except as otherwise provided in subdivision (c), for a person born in 1946 through 1952, the sum of the deductions under subsection (1)(f)(i), (ii), and (iv) is limited to $20,000.00 for a single return and $40,000.00 for a joint return. After that person reaches the age of 67, the deductions under subsection (1)(f)(i), (ii), and (iv) do not apply and that person is eligible for a deduction of $20,000.00 for a single return and $40,000.00 for a joint return, which deduction is available against all types of income and is not restricted to income from retirement or pension benefits. A person who takes the deduction under subsection (1)(e) is not eligible for the unrestricted deduction of $20,000.00 for a single return and $40,000.00 for a joint return under this subdivision.
  (e) Except as otherwise provided under subdivision (c) or (d), for a person born after 1952, the deduction under subsection (1)(f)(i), (ii), or (iv) does not apply. When that person reaches the age of 67, that person is eligible for a deduction of $20,000.00 for a single return and $40,000.00 for a joint return, which deduction is available against all types of income and is not restricted to income from retirement or pension benefits. If a person takes the deduction of $20,000.00 for a single return and $40,000.00 for a joint return, that person shall not take the deduction under subsection (1)(f)(iii) and shall not take the personal exemption under subsection (2). That person may elect not to take the deduction of $20,000.00 for a single return and $40,000.00 for a joint return and elect to take the deduction under subsection (1)(f)(iii) and the personal exemption under subsection (2) if that election would reduce that person's tax liability. A person who takes the deduction under subsection (1)(e) is not eligible for the unrestricted deduction of $20,000.00 for a single return and $40,000.00 for a joint return under this subdivision.

Reply:
Right, so did the law change each year? I'm trying to reconcile the code with the law.

Reply:
Is this addressed? Please resolve if so.

Reply:
Could you please summarize your changes for this program?

Reply:
do you mean the whole standard deduction program?